packages/jest-matchers/src/matchers.js

```diff
@@ -408,6 +408,76 @@ const matchers: MatchersObject = {
   },
 
   toMatchSnapshot,
+
+  toMatchObject(receivedObject: Object, expectedObject: Object) {
+
+    if (typeof receivedObject !== 'object' || receivedObject === null) {
+      throw new Error(
+        matcherHint('[.not].toMatch', 'object', 'expected') + '\n\n' +
+        `${RECEIVED_COLOR('string')} value must be an object.\n` +
```

Comment:
right now it prints
![screen shot 2016-11-15 at 9 33 02 pm](https://cloud.githubusercontent.com/assets/940133/20335905/2b2bfb4e-ab7b-11e6-8ead-13980d71107f.png)

you probably copied `'string'` from `.toMatch` matcher :)

i'm trying to think of a good message.

```
Expect object to be an object. Received
```
or
```
Expect expected to be an object. Received:
```
look king of weird.

maybe something like:
```
<red>object</red> value must be an object. Received:
```
and
```
<green>expected</green> value must be an object. Received:
```

thoughts?